_sass/profile.scss

```
@@ -1,5 +1,5 @@
 .profile {
-    background-color: #133667 !important;
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.05), black), no-repeat center/100% url('/assets/img/mlh-fellow.jpg'), #946c50;
```

```suggestion
    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.05), black), no-repeat center/100% url('/assets/img/background.jpg'), #946c50;
```
Renaming this to `background.jpg` would make it easier to understand and change in the future

---

Done and done, thanks!